src/discord_utils.py

```diff
@@ -31,6 +31,29 @@ async def try_set_pin(message: Message, pin_state: bool) -> None:
         print("Altering message pin state failed:", e)
 
 
+def filepath_builder(message_id: int, attachment: Attachment) -> str:
+    # Computed local filepath
+    filepath = path.join(
+        config.attachment_directory_filepath,
+        "{}.{}{}".format(
+            message_id,
+            attachment.id,
+            os.path.splitext(attachment.filename)[1],
+        ),
+    )
+    return filepath
+
+
+def is_valid_media(attachment_content_type: str) -> bool:
```

Comment:
Instead of `if (b) return false else return true,` just do `return not b`. In this case you'll should propagate the not over the rest of the condition, don't just surround it in parens and put not